pkg/subnet/byteset.go

```diff
@@ -0,0 +1,101 @@
+package subnet
```

Comment:
Perhaps a nit but could you change the name of `*byteset*.go` files to be `*byte_set*.go` just so it's easier to read?